go/libraries/doltcore/sqle/dtables/diff_table.go

```diff
@@ -383,6 +393,24 @@ func (dp diffPartition) getRowIter(ctx *sql.Context, ddb *doltdb.DoltDB, ss *sch
 	}, nil
 }
 
+func (dp *diffPartition) isDiffablePartition(ctx *sql.Context) (bool, error) {
```

Comment:
```suggestion
// isDiffablePartition checks if the commit pair for this partition is "diffable".
// If the primary key sets changed between the two commits, it may not be
// possible to diff them.
func (dp *diffPartition) isDiffablePartition(ctx *sql.Context) (bool, error) {
```